src/pages/Explore/components/Sidebar/ImageExporter/ImageError.tsx

```diff
@@ -0,0 +1,27 @@
+import { IMessageBarStyles, MessageBar, MessageBarType } from "@fluentui/react";
+import { AxiosError } from "axios";
+
+interface ImageErrorProps {
+  error: AxiosError<any, any>;
+}
+
+export const ImageError: React.FC<ImageErrorProps> = ({ error }) => {
+  if (!error) return null;
+
+  const { response } = error;
+  const errorText =
+    response?.status === 400
+      ? response?.data.error
```

Comment:
This doesn't look like the (current deployed) shape of a 400, I'm getting a JSON array of objects containing a `msg` property. Cross referenced the API PR and it doesn't look to get changed to .error there.